### DISCUSSION TITLE
rrule option missing

### DISCUSSION THREAD

**Discussion by yougotnet** [spatie/icalendar-generator#64] [Bugs]
Documentation indicates a rrule() option on the Event class, but when I install the package, I don't find it anywhere.

**Comment by rubenvanassche** [comment]
You can find it here: https://github.com/spatie/icalendar-generator/blob/master/src/Components/Event.php#L265 . RRule's were introduced with v2, maybe you're still using the old v1 version?

**Reply by cjsfj** [reply]
FWIW I had to add a `use` statement to get `Rrule` to work, which is not in the docs:

`use Spatie\IcalendarGenerator\ValueObjects\RRule;`

**Comment by cjsfj** [comment]
FWIW I had to add a `use` statement to get `RRule` to work, which is not in the docs:

`use Spatie\IcalendarGenerator\ValueObjects\RRule;`

**Reply by webleyson** [reply]
Little on google for this. Thanks.
